Src/BootCamp.Chapter/lesson3.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class lesson3
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'lesson3' to match pascal case naming rules, consider using 'Lesson3'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845544)